cmake/dependencies.cmake

```diff
@@ -7,6 +7,9 @@ pkg_check_modules(XRANDR REQUIRED xrandr)
 pkg_check_modules(XINERAMA xinerama)
 pkg_check_modules(XEXT REQUIRED xext)
 
+# for transparency support
+pkg_check_modules(XRENDER REQUIRED xrender)
```

Comment:
Does this need to be added to the NEWS section (same as you did for xft etc.)

Reply:
I've added it :)

Reply:
Thanks!